Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM alpine:3.9
+FROM alpine:3.18
 MAINTAINER Rakshit Menpara <rakshit@improwised.com>
 
-ENV DOCKERIZE_VERSION v0.6.0
+ENV DOCKERIZE_VERSION v0.7.0
```

Comment:
@codeArtisanry Move back to version 0.6.1 - We have issue on 0.7.0 https://github.com/jwilder/dockerize/issues/191

Reply:
Okay